### ISSUE TITLE
u key for update of library hangs

### ISSUE THREAD

**Issue by cpfaff** [arybczak/ncmpcpp#93]
Version of ncmpcpp 0.6.4 on Ubuntu 15.10. When pressing the `u` for update library ncmpcpp hangs. It states `ncmpcpp: Cannot send a new command while receiving another response`

**Comment by arybczak** [comment]
Please use up to date ncmpcpp version. Also, provide more information (e.g. MPD version).

**Comment by cpfaff** [comment]
He

I tried the latest version `0.7`. I use mopidy version 1.1.1

**Comment by arybczak** [comment]
What happens if you connect to mopidy via netcat (`netcat localhost 6600` assuming this is the port and host mopidy runs on) and issue an `update` command?

**Comment by cpfaff** [comment]
It says 

```
Ok mpd 0.19.0
update
Updating_db 0
Ok
```
 
Thats it.

Am 25. Dezember 2015 15:47:47 MEZ, schrieb Andrzej Rybczak <notifications@github.com>:
>What happens if you connect to mopidy via netcat (`netcat localhost
>6600` assuming this is the port and host mopidy runs on) and issue an
>`update` command?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/arybczak/ncmpcpp/issues/93#issuecomment-167243102

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by arybczak** [comment]
This is what MPD returns:

```
OK MPD 0.19.0
update
updating_db: 1
OK
```

I'm assuming that the lack of colon is the problem. Please file a mopidy bug.

**Comment by cpfaff** [comment]
I did not do a copy and paste and just forgot the colon as I was writing the answer typing manually with my mobile. Sorry for the confusion. 

```
OK MPD 0.19.0
update
updating_db: 0
OK
``

So this does not seem to be the cause of the problem.

**Comment by rytone** [comment]
same issue here - latest ncmpcpp and mopidy

**Comment by arybczak** [comment]
I do not use and hence officially support mopidy. But if someone investigated why this happens and provided a solution, it would be great.

**Comment by arybczak** [comment]
Mopidy doesn't support database updated (see #157), closing.

**Comment by arybczak** [comment]
FWIW I fixed the loop in c51b27d.

**Comment by cpfaff** [comment]
thx
